src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -50,10 +53,17 @@ const SSICredentialsRequiredScreen: FC<Props> = (props: Props): JSX.Element => {
           restrictToFormats: format,
           restrictToDIDMethods: subjectSyntaxTypesSupported,
         });
-        const matchedVCs: Array<UniqueVerifiableCredential> = selectResult.verifiableCredential
-          ? selectResult.verifiableCredential
-              .map((matchedVC: IVerifiableCredential) => getMatchingUniqueVerifiableCredential(uniqueVCs, matchedVC))
-              .filter((matchedVC): matchedVC is UniqueVerifiableCredential => !!matchedVC) // filter out the undefined (should not happen)
+        if (selectResult.areRequiredCredentialsPresent === "error") {
+          console.info('pex.selectFrom returned errors:\n', JSON.stringify(selectResult.errors));
+        }
+        const matchedVCs: Array<UniqueVerifiableCredential> = selectResult.matches && selectResult.verifiableCredential
```

Comment:
why are we checking selectResult.verifiableCredential and not using that? if it has matches it should be ok right? selectResult.matches

Reply:
The response is like this:
```
{
	"errors": [],
	"matches": [
		{
			"name": "Account",
			"rule": "all",
			"vc_path": [
				"$.verifiableCredential[0]"
			]
		}
	],
	"areRequiredCredentialsPresent": "info",
	"verifiableCredential": [
		"eyJh*snip*"
	],
	"warnings": []
}
```

Reply:
Yes and better be too defensive in cases like this